assets/avatar.svg

```diff
@@ -0,0 +1,33 @@
+                <circle id="left-eye" fill="#0F0F10" cx="40" cy="55" r="5"></circle>
+                <circle id="right-eye" fill="#0F0F10" cx="74" cy="55" r="5"></circle>
+                <g id="hoodie">
+                    <path d="M57,120 L57,127.5 L57,140 L0,140 C4.85210698,134.568461 15.1267433,129.453176 30.8239089,124.654147 C14.4289153,110.930422 0.527543937,87.0790932 0.527543937,59.248366 C0.527543937,16.8399374 31.7062842,1.3265901e-13 57,0 L57,120 L57,16 C39.2108752,16 10,30 10,67 C10,103.992483 56.9809059,119.993498 56.9999942,119.999998 L57,0 C82.2937158,1.3265901e-13 113.472456,16.8399374 113.472456,59.248366 C113.472456,87.0868268 99.5633577,110.943678 83.1624229,124.665585 C98.8632688,129.470667 109.142461,134.582138 114,140 L57,140 L57,127.5 L57,120 Z M57,16 L57,120 C57,120 104,104 104,67 C104,30 74.7891248,16 57,16 Z" fill="#E2007A"></path>
+                    <path d="M43,140 C44.2899333,134.114391 45.7769848,128.890633 48,124 L44,122 C41.5329302,127.427554 39.3685253,133.467951 38,140 L43,140 Z" id="Path-4" fill="#FEFDFC" fill-rule="nonzero"></path>
+                    <circle id="Oval" fill="#47403D" cx="46" cy="123" r="4"></circle>
+                    <path d="M71,140 C72.2899333,134.114391 73.7769848,128.890633 76,124 L72,122 C69.5329302,127.427554 67.3685253,133.467951 66,140 L71,140 Z" id="Path-4" fill="#FEFDFC" fill-rule="nonzero" transform="translate(71.000000, 131.000000) scale(-1, 1) translate(-71.000000, -131.000000) "></path>
+                    <circle id="Oval" fill="#47403D" cx="68" cy="123" r="4"></circle>
+                    <path d="M59,122 L59,140 L55,140 L55,122 C55,119.333333 59,119.333333 59,122 Z" id="zip" fill="#47403D" fill-rule="nonzero"></path>
+                </g>
+                <path d="M37.0437394,73 L76.9562606,73 C76.8965797,73.6800427 76.7764723,74.3477067 76.5999275,75 L37.4000725,75 C37.2235277,74.3477067 37.1034203,73.6800427 37.0437394,73 Z M38.6641716,78 L75.3358284,78 C74.9304945,78.6957774 74.4568218,79.3640349 73.921171,80 L40.078829,80 C39.5431782,79.3640349 39.0695055,78.6957774 38.664175,78.0000004 Z M43.402484,83 L70.597516,83 C69.5003941,83.7632167 68.2889011,84.4352403 66.9844649,85 L47.0155351,85 C45.7110989,84.4352403 44.4996059,83.7632167 43.402484,83 Z" id="mouth" fill="#8C9292"></path>
+                <g id="antenne" transform="translate(11.000000, 0.000000)">
+                    <path d="M9.23041624,46.0015296 L12,34 L9,24 L12,14 L8,5.05337691 L3,5.05337691 L7,14 L4.0273224,24 L7,34 L4.0914942,46.7193664 C5.64557471,46.2514663 7.29342081,46 9,46 C9.07692582,46 9.1537323,46.0005109 9.23041624,46.0015296 Z" id="Combined-Shape" fill="#8C9292" fill-rule="nonzero"></path>
+                    <circle id="Oval-7" fill="#B6BABA" cx="6" cy="6" r="6"></circle>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```

Comment:
I think you can reduce the file size quite a bit using https://jakearchibald.github.io/svgomg/, give it a try :)

Reply:
reduced to half size!